Backend/Backend.csproj

```diff
@@ -9,6 +9,8 @@
 	</PropertyGroup>
```

Comment:
Please review the steps in the readme.  I see it does a clone, we should be using `azd init -t ` instead.

Reply:
Also include an install link to azd in the readme.